### PR TITLE
Maintenance Day - Theme tokens refactor to semantic names

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

Swapped the theme space token syntax to use the semantic values for spaces / fontSizes ect. 

## Justify why they are needed

Maintenance Day.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx [hunk 1]
```diff
@@ -12,16 +12,16 @@ export const ScrollToButton = ({ children, type, targetRef }: ScrollToButtonProp
...
         {children}
       </Button>
-    </Wrappper>
+    </Wrapper>
   )
 }
 
-const Wrappper = styled.div({
-  paddingInline: theme.space[4],
+const Wrapper = styled.div({
```
Comment (amandaflod) [inline apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx:23]: 🤠

**Review by amandaflod — APPROVED** [review]
amazing! 🌟

File: apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply apps/store/src/components/ProductPage/ScrollToButton/ScrollToButton.tsx:23]: fun-wanna-gif-by-andrew-torralvo

**Comment by KarnellSchultz** [comment]
Trippple checked the changes here. Good to have a few more 🚀

**Pull request merged**